src/constants.ts

```diff
@@ -2,8 +2,15 @@ import { ethers } from 'ethers'
 
 export const CHI = ethers.utils.formatBytes32String('CHI').slice(0, 14)
 export const RATE = ethers.utils.formatBytes32String('RATE').slice(0, 14)
+
 export const DAI = ethers.utils.formatBytes32String('01').slice(0, 14)
 export const USDC = ethers.utils.formatBytes32String('02').slice(0, 14)
 export const ETH = ethers.utils.formatBytes32String('00').slice(0, 14)
 export const STETH = ethers.utils.formatBytes32String('05').slice(0, 14)
 export const WSTETH = ethers.utils.formatBytes32String('06').slice(0, 14)
+
+//yearn vaults 1xx
+export const YVETH = ethers.utils.formatBytes32String('100').slice(0, 14)
```

Comment:
Hmm, first of all I just realized we are keeping this list id's in (at least) 2 places -- the other one is in the [environments repo](https://github.com/yieldprotocol/environments-v2/blob/main/shared/constants.ts)  How would you like to handle this?  I don't think it's a big deal.  We can just leave it for now, maybe I'll add a comment to make sure we remember to update both places?  Alternatively we can move these constants to yield-utils or somewhere and have that be the source of truth

Reply:
Let's march environments for now, that's where the master list should live imho.